aodncore/pipeline/handlerbase.py

```diff
@@ -527,13 +528,14 @@ def logger(self):
         return self._logger
 
     @lazyproperty
-    def module_versions(self):
+    def versions(self):
```

Comment:
This isn't called explicitly anywhere?

Reply:
Nope, is only used when representing the handler as a string, i.e. in the logs